website/docs/validating-admission-policy

```diff
@@ -0,0 +1,71 @@
+---
+id: validating-admission-policy
+title: Integration with Kubernetes Validating Admission Policy
+---
+
+`Feature State`: Gatekeeper version v3.13+ (pre-alpha)
+
+> ❗ This feature is pre-alpha, subject to change (feedback is welcome!). It is disabled by default. To enable the feature,
+> set the `experimental-enable-k8s-native-validation` flag to true.
+
+## Description
+
+This feature allows Gatekeeper to integrate with Kubernetes Validating Admission Policy based on [Common Expression Language (CEL)](https://github.com/google/cel-spec), a declarative, in-process admission control alternative to validating admission webhooks.
+
+## Motivations
+
+Validating Admission Policy feature was introduced as an alpha feature to Kubernetes v1.26. Some of the benefits include 
+- in-tree/native in-process
+- reduce admission request latency
+- improve reliability and availability
+- able to fail closed without impacting availability
+- reduce operation burdens of webhooks
```

Comment:
s/reduce operation burdens of webhooks/avoid the operational burden of webhooks/